### PR TITLE
Rewrite logging config doc

### PR DESCRIPTION
**Pull request by cicdw**
Rewrites a large part of the logging configuration doc for clarity.

### CODE REVIEW

File: docs/3.0rc/develop/logging.mdx [hunk 1]
```diff
@@ -1,148 +1,64 @@
...
-As with all Prefect settings, logging settings are loaded at runtime.
-This means that to customize Prefect logging in a remote environment requires setting the appropriate environment variables and/or profile in that environment.
-</ Note>
 
 ## Prefect loggers
 
+Prefect exposes a set of loggers that you can use to emit your own custom logs.
 To use a Prefect logger, import `get_run_logger` from the `prefect.logging` module. 
 This function returns a logger instance that is aware of the current flow or task run context, allowing for more detailed and contextual logging.
+This allows you to explore logs in the UI or API based on relevant run information such as run ID and run name.
```
Comment (zzstoatzz) [inline docs/3.0rc/develop/logging.mdx:23]:
```suggestion

<Warning>
`get_run_logger()` can only be used in the context of a flow or task.

To use a normal Python logger anywhere with your same configuration, use `get_logger()` from `prefect.logging`.

The `get_logger` function will **not** send log records to the Prefect API.
</Warning>
```

**Review by zzstoatzz — APPROVED** [review]
looks good to me!

just two things:
- i think we should have some warning about where `get_run_logger` works & when it sends out records
- there's an old typo at L94: `def my_task(log_prints=False):` (can't suggest bc it wasnt the diff :/)

File: docs/3.0rc/develop/logging.mdx [hunk 2]
```diff
@@ -195,33 +111,75 @@ Using `log_prints=False` at the task level outputs:
...
-formatting for task run log records is:
+Prefect relies on [the standard Python implementation of logging configuration](https://docs.python.org/3/library/logging.config.html).
+The full specification of the default logging configuration for any version of Prefect can always be inspected [here](https://github.com/PrefectHQ/prefect/blob/main/src/prefect/logging/logging.yml).
+The default logging level is `INFO`.
 
-```python
-"%(asctime)s.%(msecs)03d | %(levelname)-7s | Task run %(task_run_name)r - %(message)s"
+### Customize logging configuration
+
+Prefect provides several settings to configure logging level and individual loggers.
```
Comment (discdiver) [inline docs/3.0rc/develop/logging.mdx:130]:
```suggestion
Prefect provides several settings to configure the logging level and individual loggers.
```

File: docs/3.0rc/develop/logging.mdx [hunk 3]
```diff
@@ -195,33 +111,75 @@ Using `log_prints=False` at the task level outputs:
...
+
+Any value in [Prefect's logging configuration file](https://github.com/PrefectHQ/prefect/blob/main/src/prefect/logging/logging.yml) can be overridden through 
+a Prefect setting of the form `PREFECT_LOGGING_[PATH]_[TO]_[KEY]=value` corresponding to the nested address of the field you are configuring.
+
+For example, to change the default logging level for flow runs but not task runs, update your profile with:
+```
```
Comment (discdiver) [inline docs/3.0rc/develop/logging.mdx:136]:
```suggestion
```bash
```

File: docs/3.0rc/develop/logging.mdx [hunk 4]
```diff
@@ -195,33 +111,75 @@ Using `log_prints=False` at the task level outputs:
...
+prefect config set PREFECT_LOGGING_LOGGERS_PREFECT_FLOW_RUNS_LEVEL="ERROR"
+```
+or set the corresponding environment variable:
 ```
```
Comment (discdiver) [inline docs/3.0rc/develop/logging.mdx:140]:
```suggestion
```bash
```

File: docs/3.0rc/develop/logging.mdx [hunk 5]
```diff
@@ -195,33 +111,75 @@ Using `log_prints=False` at the task level outputs:
...
+export PREFECT_LOGGING_LOGGERS_PREFECT_FLOW_RUNS_LEVEL="ERROR"
+```
+
+You can also configure the "root" Python logger. The root logger receives logs from all loggers unless they 
+explicitly opt out by disabling propagation. By default, the root logger is configured to output `WARNING` level logs 
+to the console. As with other logging settings, you can override this from the environment or in the logging configuration 
+file. For example, you can change the level with the `PREFECT_LOGGING_ROOT_LEVEL` environment variable .
```
Comment (discdiver) [inline docs/3.0rc/develop/logging.mdx:147]:
```suggestion
file. For example, you can change the level with the `PREFECT_LOGGING_ROOT_LEVEL` environment variable.
```

File: docs/3.0rc/develop/logging.mdx [hunk 6]
```diff
@@ -195,33 +111,75 @@ Using `log_prints=False` at the task level outputs:
...
+
+In some situations you may want to completely overhaul the Prefect logging configuration by providing your own `logging.yml` file.
+You can also create your own version of `logging.yml` in one of two ways:
```
Comment (discdiver) [inline docs/3.0rc/develop/logging.mdx:150]:
```suggestion
You can create your own version of `logging.yml` in one of two ways:
```

File: docs/3.0rc/develop/logging.mdx [hunk 7]
```diff
@@ -1,148 +1,64 @@
...
-
-![Viewing logs for a flow run in the Prefect UI](/3.0rc/img/ui/flow-run-details.png)
-
-These log messages are generated based on the configured log levels and message formatters. You can customize the log levels captured 
-(e.g. INFO, DEBUG, ERROR) and the default message format through configuration settings. 
-
-Prefect supports the standard Python logging levels `CRITICAL`, `ERROR`, `WARNING`, `INFO`, and `DEBUG`. By default,
-Prefect displays `INFO`-level and above events.
-
+When you run a flow or a task, Prefect automatically emits a standard set of logs that you can inspect in the UI or
```
Comment (discdiver) [inline docs/3.0rc/develop/logging.mdx:6]:
```suggestion
When you run a flow or a task, Prefect automatically emits a standard set of logs that you can inspect in the UI, in the CLI, or 
```

**Review by discdiver — COMMENTED** [review]
Good stuff! Minor suggestions.

**Pull request merged**